spec/lib/lightning/api_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe Lightning::Api do
+  context '.create' do
+    it 'creates the feature' do
+      Lightning::Api.create!('create_feature', 'New feature for testing')
```

Comment:
Missed a test here? I think this might work.

```suggestion
      Lightning::Api.create!('create_feature', 'New feature for testing')
      expect(Lightning::Api.get('create_feature')).to_not be_empty
```